build.gradle

```diff
@@ -37,7 +37,7 @@ def versions = [
         reformS2sClient    : '3.1.1',
         serenity           : '2.0.76',
         sonarPitest        : '0.5',
-        springBoot         : '2.1.4.RELEASE',
+        springBoot         : '2.3.3.RELEASE',
         springHystrix      : '2.1.1.RELEASE',
```

Comment:
can we also make change to use /health/readiness endpoint instead of /health ?

Reply:
Upgrade to chart-java 3.0 which uses /health/readiness

Reply:
This also requires moving applicationInsightsInstrumentKey to secrets from charts